config/locals.js

```diff
@@ -40,6 +44,46 @@ function cacheCampaign(phoenixCampaign) {
 }
 
 /**
+ * Create Doing/Completed Mobile Commons Groups to support Mobile Commons broadcasting.
+ * @see https://github.com/DoSomething/gambit/issues/673
+ */
+function createMobileCommonsGroupsForCampaign(campaignModel) {
```

Comment:
I started on this function before figuring out how to use Mongoose instance method (implemented #675). This would be a perfect candidate to add as an [instance method](http://mongoosejs.com/docs/guide.html#methods) to our `Campaign` model .